pypeit/core/save.py

```diff
@@ -399,47 +399,49 @@ def save_1d_spectra_fits(specobjs, header, outfile, helio_dict=None, telescope=N
     # Loop on detectors
     npix = 0
     ext = 0
-    for specobj in all_specobj:
-        if specobj is None:
+    for spec in specobjs:
+        if spec is None:
```

Comment:
Using ```spec``` here is a bit funny.  I should prefer ```sobj```
But I'll live with it as is.

Reply:
Fixed to sobj.